src/migrations/data.ts

```diff
@@ -555,3 +595,14 @@ export function createEmptyExcelWorkbookData(): ExcelWorkbookData {
     sheets: [createEmptyExcelSheet(INITIAL_SHEET_ID, "Sheet1")],
   };
 }
+
+function getDuplicateSheetName(nameToDuplicate: string, names: string[]): string {
+  let i = 1;
+  const baseName = `Copy of ${nameToDuplicate}`;
```

Comment:
To avoid breaking existing spreadsheets, we should preserve the previous behavior (even if it's not correct in a collaborative context).
Let's say a company has all users in FR, they used to have a sheet named "Copie de [...]" but now it's "Copy of [...]" and they did based on the sheet name is broken (formulas)
```suggestion
  const baseName = _lt("Copy of %s", nameToDuplicate);
```

Reply:
Yeah but that means that after repairing the revisions, the commands will be different for each user based on it's language. Isn't it way more of a problem ?

Reply:
I think it's better to keep the current problem as it is because:

- we didn't get any ticket or feedback
- If this bug caused an issue for some customer, it is very likely that they renamed the sheet to make it work again.

so in practice I don't think it's a real issue in existing spreadsheets (or they have been solved already by renaming)

On the other hand, if we change the behavior, there's a risk of breaking spreadsheets that are working right now.